### DISCUSSION TITLE
HTML Image File Upload

### DISCUSSION THREAD

**Discussion by WEBentist** [elementor/elementor#23127] [Developers Community]
Is there any possibility to upload HTML Image in elementor

https://prnt.sc/1hKIjyDM9KKc

https://prnt.sc/ybjpkAfAezGi

**Comment by hitsgee** [comment]
HTML is code not sure why you have this as an image. SVG images are supported in elementor along with Json (Lottie) this option needs to be enabled in settings though (unfiltered files).

If you open that html file on your PC does it contain an image? If so can you not just right click and save it?

**Reply by WEBentist** [reply]
This is the generated as one of the download options, i will share the original file and the source file if need be
